src/identity-providers/add/AddIdentityProvider.tsx

```diff
@@ -40,14 +38,14 @@ export const AddIdentityProvider = () => {
     try {
       await adminClient.identityProviders.create({
         ...provider,
-        providerId: providerId,
+        providerId,
         alias: providerId,
       });
       addAlert(t("createSuccess"), AlertVariant.success);
       history.push(
         toIdentityProvider({
           realm,
-          providerId: providerId,
+          providerId,
```

Comment:
Once `providerId` is required you can use the non-null assertion operator here.
```suggestion
          providerId: providerId!,
```